.github/workflows/ci.yml

```
@@ -11,11 +11,11 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - elixir_version: 1.15.4
-            otp_version: 26.0.2
+          - elixir_version: 1.16.1
+            otp_version: 26.2.2
             lint: true
-          - elixir_version: 1.14.5
-            otp_version: 25.3.2.5
+          - elixir_version: 1.15.7
+            otp_version: 25.3.2.9
```

The idea is to run on the latest and oldest supported version :)

```suggestion
          - elixir_version: 1.14.5
            otp_version: 25.3.2.5
```

---

right, 1.14 still listed in the `mix.exs`, good call